src/inventory_ui.cpp

```diff
@@ -275,7 +275,7 @@ void inventory_selector_preset::append_cell( const
 }
 
 void inventory_selector_preset::append_cell( const
-        std::function<std::string( const inventory_entry & )> &func,
+        std::function<std::string( const inventory_entry & )> &getTextFunc,
```

Comment:
An unrelated change which violates the [code style guide](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.md).

Reply:
Fixed - reverted